app_build_suite/__main__.py

```diff
@@ -73,11 +76,25 @@ def configure_global_options(config_parser: configargparse.ArgParser):
     )
 
 
+def get_default_config_file_path() -> str:
+    # this is the only place where we check for command line option directly,
+    # as that's the only way to change where we load the file from
+    # FIXME: it's also hacky, as it relies on helm pipeline to provide the "-c" option
+    opt = "-c"
+    base_dir = ""
+    if opt in sys.argv:
+        c_ind = sys.argv.index(opt)
+        base_dir = sys.argv[c_ind + 1]
+    config_path = os.path.join(base_dir, ".abs", "main.yaml")
+    return config_path
```

Comment:
What about adding a separate `--config-path` flag?

Reply:
It's a good idea, but I think it's a bit orthogonal. Overloading would be good, but still autodetection should match the basic chart layout scenario people have:
- 1 chart per repo in main repo dir
- N charts per repo in subdirs

This hack covers the 2nd case. Currently, I don't have any fast way of solving it better.